lib/util/semver.js

```diff
@@ -190,12 +190,19 @@ exports.parseRange = str => {
 		// eslint-disable-next-line no-sparse-arrays
 		return [, ...arr, ...items.slice(1).map(() => fn)];
 	};
+	// remove whitespace characters after ^, ~, <=, <, >=, >, =
+	const trimRangeOperators = str => {
+		return str.replace(/(\^|~|<=|<|>=|>|=)\s*/g, "$1");
+	};
```

Comment:
@sokra for your convenience, here is the open discussion from the previous PR:

> **sokra** 
> I think that's better fixed in `parseSimple` by adding whitespace to the regexp there 

> **nikmilson**
> Yeah, I'd be happy to rather do it there, as well. But it won't fix the issue unfortunately, since it happens before `parseSimple`. E. g. `'>= 16.8.0'.trim().split(/\s+/g)` (old code) will produce `[ '>=', '16.8.0' ]` and pass each element to parseSimple, thus it's already too late to take care about trimming.